packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -201,11 +209,13 @@ Media.createMediaConnection = (
       },
     },
     {
+      // TODO: RoapMediaConnection is not ready to use stream classes yet, so we pass the raw MediaStreamTrack for now
       localTracks: {
-        audio: audioTrack?.underlyingTrack,
-        video: videoTrack?.underlyingTrack,
-        screenShareVideo: shareVideoTrack?.underlyingTrack,
-      },
+        audio: audioStream?.outputStream.getTracks()[0],
+        video: videoStream?.outputStream.getTracks()[0],
+        screenShareVideo: shareVideoStream?.outputStream.getTracks()[0],
+        screenShareAudio: shareAudioStream?.outputStream.getTracks()[0], // TODO: add type for screenShareAudio in internal-media-core
```

Comment:
again, TODO without a jira - this one should reference https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-446923 and same todo comment is twice in 2 consecutive lines, shouldn't one be enough?

Reply:
Looks like instead of moving it to next line, I've copy pasted. Will remove one of them and add the Jira

Reply:
Done in #3053